app/src/main/java/com/kickstarter/ui/viewholders/discoverydrawer/TopFilterViewHolder.kt

```diff
@@ -0,0 +1,53 @@
+package com.kickstarter.ui.viewholders.discoverydrawer
+
+import androidx.core.content.res.ResourcesCompat
+import com.kickstarter.R
+import com.kickstarter.databinding.DiscoveryDrawerTopFilterViewBinding
+import com.kickstarter.libs.utils.ObjectUtils
+import com.kickstarter.ui.adapters.data.NavigationDrawerData
+import com.kickstarter.ui.viewholders.KSViewHolder
+
+class TopFilterViewHolder(
+    private val binding: DiscoveryDrawerTopFilterViewBinding,
+    private val delegate: Delegate
+) : KSViewHolder(binding.root) {
+    private var item: NavigationDrawerData.Section.Row? = null
+
+    interface Delegate {
+        fun topFilterViewHolderRowClick(viewHolder: TopFilterViewHolder, row: NavigationDrawerData.Section.Row)
+    }
+
+    @Throws(Exception::class)
+    override fun bindData(data: Any?) {
+        item = ObjectUtils.requireNonNull(data as NavigationDrawerData.Section.Row?, NavigationDrawerData.Section.Row::class.java)
+    }
+
+    override fun onBind() {
+        val context = context()
+
+        val textColor = if (item?.selected() == true) context.resources.getColor(R.color.accent, null)
```

Comment:
I'm usually not picky with the syntax, as we do not have an automatic linter in place yet, but this is confusing to read as it is, could you change it to something like this? 
![Uploading Screen Shot 2021-02-09 at 3.38.44 PM.png…]()